courses/models.py

```diff
@@ -90,7 +90,6 @@ class Program(TimestampedModel, ValidateOnSaveMixin):
         max_length=255, unique=True, validators=[validate_url_path_field]
     )
     live = models.BooleanField(default=False)
-    # products = GenericRelation(Product, related_query_name="programs")
```

Comment:
Should this line be removed?

Reply:
That's been commented out for a while now so I pulled it. (I don't think that GenericRelation can work since pulling in the Products model will create a circular import.)

Reply:
GenericRelation should accept a string argument for the model name I think.